app/components/header/view.html.erb

```diff
@@ -23,12 +23,10 @@
         <nav>
           <ul id="navigation" class="govuk-header__navigation govuk-header__navigation--end" aria-label="Top Level Navigation">
             <li class="govuk-header__navigation-item">
-              <%# TODO Update the path once it has been created %>
-              <%= govuk_link_to "Your account", "#2", class: "govuk-header__link" %>
+              <%= govuk_link_to "Your account", Settings.dfe_sign_in.profile, class: "govuk-header__link" %>
```

Comment:
Just checked with @edwardhorsford and we should remove this link for now. It wasn't intended to point to sign in and we don't have anything else for it just yet.

Reply:
removed